src/muya/lib/config.js

```diff
@@ -6,7 +6,7 @@ import voidHtmlTags from 'html-tags/void'
 // Electron 2.0.2 not support yet! So give a default value 4
 export const DEVICE_MEMORY = navigator.deviceMemory || 4 // Get the divice memory number(Chrome >= 63)
 export const UNDO_DEPTH = DEVICE_MEMORY >= 4 ? 100 : 50
-export const HAS_TEXT_BLOCK_REG = /^(h\d|span|th|td|hr|pre)/i
+export const HAS_TEXT_BLOCK_REG = /^(h\d|span|th|td|hr|pre|li|ul)/i
```

Comment:
`li` and `ul` should not has `text` in these blocks.

Reply:
@Jocs Seems that this caused by empty list.so I will close this PR.

Reply:
@enyaxu Does this issue fixed?

Reply:
Yes, it's fixed with my PR.

Reply:
😄 👍